holder.yml

```diff
@@ -10,10 +10,106 @@ info:
     name: GitHub Source Code
     url: https://github.com/w3c-ccg/vc-api
 paths:
+  /credentials/{id}:
+    get:
+      tags:
+       - Credentials
+      summary: Gets a credential or verifiable credential by ID
+      operationId: getCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Credential retrieved
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: "./components/Credential.yml#/components/schemas/Credential"
+                  - $ref: "./components/VerifiableCredential.yml#/components/schemas/VerifiableCredential"
+        "400":
+          description: Bad Request
+        "401":
+          description: Not Authorized
+        "410":
+          description: Gone! There is no data here
+        "418":
+          description: I'm a teapot - MUST not be returned outside of pre-arranged scenarios between both parties
+        "500":
+          description: Internal Error
+        "501":
+          description: Not Implemented
+    delete:
+      tags:
+       - Credentials
+      summary: Deletes a credential or verifiable credential by ID
+      operationId: deleteCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "202":
+          description: Credential deleted - this is a 202 by default as soft deletes and processing time are assumed
```

Comment:
This should probably say "Credential marked for deletion". Looks like the proposal here is that this call is sometimes implemented as eventually consistent.